src/Console/Command/PurgeCommand.php

```diff
@@ -39,8 +39,9 @@ protected function configure()
 newest json file in the include directory of the given output-dir.
 
 In your satis.json (or other name you give), you must define
-"archive" argument.
+"archive" argument. You also need to define "homepage"* if you don't use archive "prefix-url".
 
+* "homepage" may be overloaded using SATIS_HOMEPAGE environment variable.
```

Comment:
I don't have access to edit the original repo anymore, but that would be the following change:

```suggestion
"archive" argument. You also need to define "homepage" argument or "SATIS_HOMEPAGE" environment variable if you don't use archive "prefix-url" argument.
```

Reply:
Ah wait, I can't commit those suggested changes either cause they would be applied to the remote branch in a repository that I have no access to.